Dockerfile.multichain

```diff
@@ -19,9 +19,19 @@ RUN sed -i 's#target-dir = "../target"#target-dir = "target"#' .cargo/config.tom
 RUN cargo build --release --package mpc-node
 
 FROM debian:stable-slim as runtime
-RUN apt-get update && apt-get install --assume-yes libssl-dev ca-certificates curl
+RUN apt-get update && apt-get install --assume-yes libssl-dev ca-certificates curl redis-server
+
 RUN update-ca-certificates
 COPY --from=builder /usr/src/app/target/release/mpc-node /usr/local/bin/mpc-node
+
+# Create a script to start both Redis and the Rust app
+RUN echo "#!/bin/bash\nservice redis-server start\nexec mpc-node" > /start.sh \
```

Comment:
I think we may need to point to the location of the config file in order for it to not start in default config mode.

https://stackoverflow.com/a/23498203